docs/topics/ref_proficiency-levels.adoc

```diff
@@ -2,23 +2,22 @@
 [id='proficiency-levels']
 = Proficiency levels
 
-Each available mission teaches concepts that require certain minimum knowledge. This requirement varies by mission. The minimum requirements and concepts are organized in several levels of proficiency. In addition to the levels described here, you might need additional information specific to each mission.
+Each available example teaches concepts that require certain minimum knowledge. This requirement varies by example. The minimum requirements and concepts are organized in several levels of proficiency. In addition to the levels described here, you might need additional information specific to each example.
 
 [discrete]
 [[proficiency_foundational]]
 == {proficiency-foundational}
 
-The missions rated at {proficiency-foundational} proficiency generally require no prior knowledge of the subject matter; they provide general awareness and demonstration of key elements, concepts, and terminology. There are no special requirements except those directly mentioned in the description of the mission.
+The examples rated at {proficiency-foundational} proficiency generally require no prior knowledge of the subject matter; they provide general awareness and demonstration of key elements, concepts, and terminology. There are no special requirements except those directly mentioned in the description of the example.
 
 [discrete]
 [[proficiency_advanced]]
 == {proficiency-advanced}
 
-When using {proficiency-advanced} missions, the assumption is that you are familiar with the common concepts and terminology of the subject area of the mission in addition to Kubernetes and OpenShift. You must also be able to perform basic tasks on your own, for example configure services and applications, or administer networks. If a service is needed by the mission, but configuring it is not in the scope of the mission, the assumption is that you have the knowledge to to properly configure it, and only the resulting state of the service is described in the documentation.
+When using {proficiency-advanced} examples, the assumption is that you are familiar with the common concepts and terminology of the subject area of the example in addition to Kubernetes and OpenShift. You must also be able to perform basic tasks on your own, for example configure services and applications, or administer networks. If a service is needed by the example, but configuring it is not in the scope of the example, the assumption is that you have the knowledge to to properly configure it, and only the resulting state of the service is described in the documentation.
```

Comment:
for example configure services and applications, or administer networks.--> for example, configuring services and applications, or administering networks.
Repeated "to" in the last sentence: you have the knowledge to to properly configure it